sources/packages/web/src/components/generic/StatusChipBypass.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <chip-status :status="chipStatus" :label="bypassStatus" />
+</template>
+<script lang="ts">
+import { computed, defineComponent, PropType } from "vue";
+import ChipStatus from "@/components/generic/ChipStatus.vue";
+import { ApplicationRestrictionBypassStatus } from "@/services/http/dto";
+import { useBypassStatus } from "@/composables";
+
+export default defineComponent({
+  components: { ChipStatus },
+  props: {
+    isActive: {
+      type: Boolean as PropType<boolean>,
+      required: true,
+    },
+  },
+  setup(props) {
+    const { mapBypassStatus } = useBypassStatus();
+    const bypassStatus = props.isActive
+      ? ApplicationRestrictionBypassStatus.Active
+      : ApplicationRestrictionBypassStatus.Removed;
+    const chipStatus = computed(() => mapBypassStatus(bypassStatus));
```

Comment:
IMO the `mapBypassStatus` must happen based on raw status not the translated label name. like this. 

```ts
export function useBypassStatus() {
  const mapBypassStatus = (
    restrictionBypassStatus: boolean,
  ): StatusChipTypes => {
    return restrictionBypassStatus
      ? StatusChipTypes.Warning
      : StatusChipTypes.Success;
  };

  return { mapBypassStatus };
}
```

Also the warning and success should exactly replicate the pattern of restrictions. (The code snipper above reflects this logic)

i,e when active  - Warning
when removed - Success

![image](https://github.com/user-attachments/assets/4164db8d-eafe-4efc-9b20-18a1297822f6)

Reply:
Updated.Updated.